src/main/java/com/treasuredata/client/model/TDTable.java

```diff
@@ -37,6 +37,8 @@ public class TDTable
     private final String expireDays;
     private final String createdAt;
     private final String updatedAt;
+    private final long userId;
```

Comment:
In td-client-java, `user_id` is handled as an integer value as follows:
https://github.com/treasure-data/td-client-java/blob/75366cdfe7eba1018aeb4d253ba045fb905cebc8/src/main/java/com/treasuredata/client/model/TDApiKey.java#L16

It's good to follow the existing data type for `user_id` as much as possible.